cmd/protoc-gen-fastmarshal/templates/fieldsnippets.tmpl

```diff
@@ -855,6 +855,62 @@
             m.{{.GoName}} = {{if eq $syntax "proto2"}}csproto.Int{{$bitSize}}(v){{else}}v{{end}}
         }
     {{- end -}}
+{{- else if eq $kind "float" -}}
+    {{- if eq $cardinality "repeated" -}}
+        switch wt {
+        case csproto.WireTypeVarint:
+            if v, err := dec.DecodeFloat32(); err != nil {
+                return fmt.Errorf("unable to decode float32 value for field '{{.Desc.Name}}' (tag={{.Desc.Number}}): %w", err)
+            } else {
+                m.{{.GoName}} = append(m.{{.GoName}}, v)
+            }
+        case csproto.WireTypeLengthDelimited:
```

Comment:
I'll have to go check the docs, but I think this case is unnecessary.  Repeated integer fields in proto3 use packed encoding by default but I believe that's not the case for `float` and `double`.